src/mapbox/CountyLayer.js

```diff
@@ -1,7 +1,9 @@
-import React from 'react'
+import React, { useMemo } from 'react'
```

Comment:
Does it make sense to move this logic into a separate component e.g. CountySnapshotOverlay... or maybe even combine it with the CountySnapshotOverlay?